### ISSUE TITLE
Pipeline not working on dev branch

### ISSUE THREAD

**Issue by JihedC** [JihedC/Snakemake_ATAC_2020#11]
Hi Jianhui,

I was trying to run the pipeline on shark but I have got this error message:
'''
Start time : Wed Mar 17 14:35:21 CET 2021
/exports/humgen/jihed/miniconda3/lib/python3.6/site-packages/snakemake/workflow.py:26: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\t'.
  from snakemake.parser import parse
Building DAG of jobs...
Creating conda environment envs/homer.yaml...
Downloading remote packages.
CreateCondaEnvironmentException:
Could not create conda environment from /exports/humgen/jihed/Jianhui/Snakemake_ATAC_2020/ATAC-seq_pipeline/rules/../envs/homer.yaml:
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed

ResolvePackageNotFound:
  - hmmratac=4.11

End time : Wed Mar 17 14:35:39 CET 2021
'''

I think there is a mistake with the version of hmmratac. On conda I can find only hmmratac 1.2.10.

Can you fix this please?

**Comment by WJH58** [comment]
@JihedC Isn't there problem with homer? Not hmmratac.

**Comment by JihedC** [comment]
the homer environment contain hmmratac for some reasons.
https://github.com/JihedC/Snakemake_ATAC_2020/blob/dev/ATAC-seq_pipeline/envs/homer.yaml

**Comment by WJH58** [comment]
@JihedC Since we wrote homer documentation so I deleted homer.yaml and homer.smk. There should be no problem now.

**Comment by JihedC** [comment]
Ok I'll test it again this week

**Comment by JihedC** [comment]
The pipeline works on the dev branch, there are some issues with the peak calling but I think it's because the bam files are too small in the test datasets.

**Comment by WJH58** [comment]
@JihedC Yes exactly. Test bam files are not paired.

**Comment by JihedC** [comment]
sorry I meant peak calling with MACS2.
And I thought you changed the mapping parameters now to only keep the paired files?

**Comment by WJH58** [comment]
@JihedC Yes I have changed the mapping rule and it will produce 100% paired reads.

**Comment by JihedC** [comment]
Then why do you say that the test bam files are not paired ?
